src/mainTemplate.json

```diff
@@ -1911,7 +1918,10 @@
       "No": "empty/empty-kibana-ip-resources.json",
       "Yes": "ips/kibana-ip-resources.json"
     },
-    "kibanaIpTemplateUrl": "[concat(variables('templateBaseUrl'), variables('kibanaIpTemplates')[variables('topologySettings').kibana])]"
+    "kibanaIpTemplateUrl": "[concat(variables('templateBaseUrl'), variables('kibanaIpTemplates')[variables('topologySettings').kibana])]",
+    "monitoringSettings":{
```

Comment:
This also needs to be added at the bottom of the main template in 'resources -> properties -> parameters' section. Must be:
```
"monitoringSettings": {
  "value": "[variables('monitoringSettings')]"
}
```